### ISSUE TITLE
Clearing watch memory

### ISSUE THREAD

**Issue by eithrial** [cmaion/polar#7]
Is there possibility to add option to remove history from Polar watch after syncing using 'polar_ftp sync'?

**Comment by eithrial** [comment]
I do not use Polar Flow and I would like to avoid sending there any pieces of information while changing settings on my watch.
Thanks for pointing Antash fork.

**Comment by cmaion** [comment]
Are you sure that the watch doesn't automatically clears older records by itself, without any help from Polar Flow?

**Comment by eithrial** [comment]
No, it removes old data. But I do not want to share them with Polar which is sometimes unavoidable when settings are being changed.

**Comment by carmenfdezb** [comment]
@eithrial, I've updated Antash fork with @cmaion lasts commits, you can get it in a new branch that I've created in my fork: https://github.com/carmenfdezb/polar/tree/Antash-master

I've tested removing history data and works fine (well, I get some errors, but finally I get to remove data).
